lib/libutils/isoc/include/malloc.h

```diff
@@ -30,15 +30,20 @@
 #include <stddef.h>
 #include <types_ext.h>
 
+#if defined(CFG_TEE_CORE_MALLOC_DEBUG)
```

Comment:
This is a publicly exported file, I don't like very much the fact that we put a TEE core private configuration flag here. `core/core.mk` may be a better place.